ajax/libs/swiftclick/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "swiftclick",
+  "description": "Eliminates the 300ms click event delay on touch devices that support orientation change",
+  "author": {
+    "name": "Ivan Hayes",
+    "url": "http://www.ivanhayes.com"
+  },
+  "homepage": "https://github.com/munkychop/swiftclick",
+  "keywords": [
+    "swiftclick",
+    "fastclick",
+    "mobile",
+    "tap",
+    "touch",
+    "click",
+    "delay"
+  ],
+  "filename": "swiftclick.min.js",
+  "version": "1.4.1",
+  "autoupdate": {
```

Comment:
We prefer to use [npm auto-update config](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md) if the package has been published on npm, but I found that the file structure on GitHub repo is different from npm. Would you like to discuss with author if he can publish `dist` folder on npm also? Thanks.